drivers/display/ssd16xx.c

```diff
@@ -569,7 +572,7 @@ static int ssd16xx_init(struct device *dev)
 	}
 
 	gpio_pin_configure(driver->reset, SSD16XX_RESET_PIN,
-			   GPIO_DIR_OUT);
+			   GPIO_OUTPUT_HIGH | SSD16XX_RESET_FLAGS);
```

Comment:
Since we want the driver to accept also `GPIO_ACTIVE_HIGH` flag for the pin it's better to use `GPIO_OUTPUT_INACTIVE` rather than `GPIO_OUTPUT_HIGH`.